app/assets/stylesheets/darkswarm/shop.css.scss

```diff
@@ -125,21 +125,44 @@
     }
   }
 
-  .shopfront_closed_message, .shopfront_hidden_message {
-    padding: 15px;
-    border-radius: 5px;
-  }
+  .closed_shop_header {
+    background-color: $grey-650;
+    color: $white;
 
-  .shopfront_closed_message {
-    border: 2px solid #eb4c46;
-  }
+    h4 {
+      color: $white;
+
+      div {
+        display: inline-block;
+      }
+    }
 
-  .shopfront_closed_message {
-    margin: 2em 0em;
+    p {
+      line-height: 2.5rem;
+      margin-bottom: 0;
+    }
   }
 
-  .shopfront_hidden_message {
-    border: 2px solid #db4;
-    margin: 2em 0em;
+  .warning_sign {
+    margin: 0 10px 0 5px;
+
+    .rectangle {
+      position: absolute;
+      top: 27px;
+      color: $grey-650;
+      width: 23px;
+      height: 23px;
+      transform: rotate(-315deg);
+      border-radius: 4px;
+      background-color: #ffffff;
+    }
+
+    strong {
+      text-align: center;
+      color: $grey-650;
+      display: block;
+      position: relative;
+      width: 23px;
+    }
```

Comment:
I saw the warning sign done with css in zeplin so I gave it a try. It's not a beauty but acceptable?
![image](https://user-images.githubusercontent.com/1640378/79856172-e4e07c80-83c3-11ea-9d17-8984527069c7.png)

Reply:
isn't there an icon from font-awesome that we could use? it seems a bit more maintainable to me given we already import it.

Reply:
I dont think fontawesome is used in the frontoffice...
I spent one more hour on this without success so I am leaving as it is...

Reply:
Who knows what mess we might have between front/backoffice assets...